Startup fails for Exasol DB version 8.25.0

When using Exasol Docker DB version 8.25.0, constructor call `new ExasolTestcontainerTestSetup()` fails with the following exception:
```
java.lang.IllegalStateException: E-ETAJ-12: Failed to ssh to the exasol database. This is required for redirecting a host port. Known mitigations:
* Make sure the database is reachable, i.e. port 32784 open on host '192.168.56.6'.
* Make sure user 'root' can login to the database using your ssh-key.
 at com.exasol.exasoltestsetup.SshConnection.createSession([SshConnection.java:47](vscode-file://vscode-app/Users/chp/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/code/electron-sandbox/workbench/workbench.html))
 at com.exasol.exasoltestsetup.SshConnection.([SshConnection.java:31](vscode-file://vscode-app/Users/chp/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/code/electron-sandbox/workbench/workbench.html))
 at com.exasol.exasoltestsetup.testcontainers.ExasolTestcontainerTestSetup.([ExasolTestcontainerTestSetup.java:43](vscode-file://vscode-app/Users/chp/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/code/electron-sandbox/workbench/workbench.html))
 at com.exasol.exasoltestsetup.testcontainers.ExasolTestcontainerTestSetupIT.getExasolTestSetup([ExasolTestcontainerTestSetupIT.java:10](vscode-file://vscode-app/Users/chp/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/code/electron-sandbox/workbench/workbench.html))
 at com.exasol.exasoltestsetup.ExasolTestSetupTestBase.beforeEach([ExasolTestSetupTestBase.java:41](vscode-file://vscode-app/Users/chp/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/code/electron-sandbox/workbench/workbench.html))
 at java.base/java.util.ArrayList.forEach([ArrayList.java:1541](vscode-file://vscode-app/Users/chp/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/code/electron-sandbox/workbench/workbench.html))
 at java.base/java.util.ArrayList.forEach([ArrayList.java:1541](vscode-file://vscode-app/Users/chp/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/code/electron-sandbox/workbench/workbench.html))
Caused by: com.jcraft.jsch.JSchException: Algorithm negotiation fail
 at com.jcraft.jsch.Session.receive_kexinit([Session.java:590](vscode-file://vscode-app/Users/chp/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/code/electron-sandbox/workbench/workbench.html))
 at com.jcraft.jsch.Session.connect([Session.java:320](vscode-file://vscode-app/Users/chp/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/code/electron-sandbox/workbench/workbench.html))
 at com.jcraft.jsch.Session.connect([Session.java:183](vscode-file://vscode-app/Users/chp/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/code/electron-sandbox/workbench/workbench.html))
 at com.exasol.exasoltestsetup.SshConnection.createSession([SshConnection.java:39](vscode-file://vscode-app/Users/chp/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/code/electron-sandbox/workbench/workbench.html))
 ... 7 more
```

## Workaround
Use Exasol DB 8.24.0

After enabling log level debug for the ssh server it logs the following error:

```
Apr 15 14:44:42 n11 sshd[12282]: Unable to negotiate with 172.17.0.1 port 60756: no matching host key type found. Their offer: ssh-rsa,ssh-dss,ecdsa-sha2-nistp256,ecdsa-sha2-nistp384,ecdsa-sha2-nistp521 [preauth]
```

<details>
<summary>Complete log</summary>
<pre>
Apr 15 14:44:42 n11 sshd[2810]: debug3: fd 5 is not O_NONBLOCK
Apr 15 14:44:42 n11 sshd[2810]: debug1: Forked child 12282.
Apr 15 14:44:42 n11 sshd[2810]: debug3: send_rexec_state: entering fd = 8 config len 3256
Apr 15 14:44:42 n11 sshd[2810]: debug3: ssh_msg_send: type 0
Apr 15 14:44:42 n11 sshd[2810]: debug3: send_rexec_state: done
Apr 15 14:44:42 n11 sshd[12282]: debug3: oom_adjust_restore
Apr 15 14:44:42 n11 sshd[12282]: debug1: Set /proc/self/oom_score_adj to 0
Apr 15 14:44:42 n11 sshd[12282]: debug1: rexec start in 5 out 5 newsock 5 pipe 7 sock 8
Apr 15 14:44:42 n11 sshd[12282]: debug1: inetd sockets after dupping: 4, 4
Apr 15 14:44:42 n11 sshd[12282]: Connection from 172.17.0.1 port 60756 on 172.17.0.3 port 22 rdomain ""
Apr 15 14:44:42 n11 sshd[12282]: debug1: Local version string SSH-2.0-OpenSSH_8.9p1 Ubuntu-3ubuntu0.6
Apr 15 14:44:42 n11 sshd[12282]: debug1: Remote protocol version 2.0, remote software version JSCH-0.1.54
Apr 15 14:44:42 n11 sshd[12282]: debug1: compat_banner: no match: JSCH-0.1.54
Apr 15 14:44:42 n11 sshd[12282]: debug2: fd 4 setting O_NONBLOCK
Apr 15 14:44:42 n11 sshd[12282]: debug3: ssh_sandbox_init: preparing seccomp filter sandbox
Apr 15 14:44:42 n11 sshd[12282]: debug2: Network child is on pid 12283
Apr 15 14:44:42 n11 sshd[12282]: debug3: preauth child monitor started
Apr 15 14:44:42 n11 sshd[12282]: debug3: privsep user:group 102:65534 [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug1: permanently_set_uid: 102/65534 [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug3: ssh_sandbox_child: setting PR_SET_NO_NEW_PRIVS [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug3: ssh_sandbox_child: attaching seccomp filter program [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug3: append_hostkey_type: ssh-rsa key not permitted by HostkeyAlgorithms [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug1: list_hostkey_types: rsa-sha2-512,rsa-sha2-256 [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug3: send packet: type 20 [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug1: SSH2_MSG_KEXINIT sent [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug3: receive packet: type 20 [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug1: SSH2_MSG_KEXINIT received [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug2: local server KEXINIT proposal [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug2: KEX algorithms: curve25519-sha256,curve25519-sha256@libssh.org,ecdh-sha2-nistp256,ecdh-sha2-nistp384,ecdh-sha2-nistp521,sntrup761x25519-sha512@openssh.com,diffie-hellman-group-exchange-sha256,diffie-hellman-group16-sha512,diffie-hellman-group18-sha512,diffie-hellman-group14-sha256,kex-strict-s-v00@openssh.com [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug2: host key algorithms: rsa-sha2-512,rsa-sha2-256 [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug2: ciphers ctos: aes128-ctr,aes192-ctr,aes256-ctr,aes128-gcm@openssh.com,aes256-gcm@openssh.com [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug2: ciphers stoc: aes128-ctr,aes192-ctr,aes256-ctr,aes128-gcm@openssh.com,aes256-gcm@openssh.com [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug2: MACs ctos: umac-64-etm@openssh.com,umac-128-etm@openssh.com,hmac-sha2-256-etm@openssh.com,hmac-sha2-512-etm@openssh.com,hmac-sha1-etm@openssh.com,umac-64@openssh.com,umac-128@openssh.com,hmac-sha2-256,hmac-sha2-512,hmac-sha1 [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug2: MACs stoc: umac-64-etm@openssh.com,umac-128-etm@openssh.com,hmac-sha2-256-etm@openssh.com,hmac-sha2-512-etm@openssh.com,hmac-sha1-etm@openssh.com,umac-64@openssh.com,umac-128@openssh.com,hmac-sha2-256,hmac-sha2-512,hmac-sha1 [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug2: compression ctos: none,zlib@openssh.com [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug2: compression stoc: none,zlib@openssh.com [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug2: languages ctos:  [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug2: languages stoc:  [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug2: first_kex_follows 0  [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug2: reserved 0  [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug2: peer client KEXINIT proposal [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug2: KEX algorithms: ecdh-sha2-nistp256,ecdh-sha2-nistp384,ecdh-sha2-nistp521,diffie-hellman-group14-sha1,diffie-hellman-group-exchange-sha256,diffie-hellman-group-exchange-sha1,diffie-hellman-group1-sha1 [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug2: host key algorithms: ssh-rsa,ssh-dss,ecdsa-sha2-nistp256,ecdsa-sha2-nistp384,ecdsa-sha2-nistp521 [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug2: ciphers ctos: aes128-ctr,aes128-cbc,3des-ctr,3des-cbc,blowfish-cbc,aes192-ctr,aes192-cbc,aes256-ctr,aes256-cbc [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug2: ciphers stoc: aes128-ctr,aes128-cbc,3des-ctr,3des-cbc,blowfish-cbc,aes192-ctr,aes192-cbc,aes256-ctr,aes256-cbc [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug2: MACs ctos: hmac-md5,hmac-sha1,hmac-sha2-256,hmac-sha1-96,hmac-md5-96 [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug2: MACs stoc: hmac-md5,hmac-sha1,hmac-sha2-256,hmac-sha1-96,hmac-md5-96 [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug2: compression ctos: none [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug2: compression stoc: none [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug2: languages ctos:  [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug2: languages stoc:  [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug2: first_kex_follows 0  [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug2: reserved 0  [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug1: kex: algorithm: ecdh-sha2-nistp256 [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug1: kex: host key algorithm: (no match) [preauth]
Apr 15 14:44:42 n11 sshd[12282]: Unable to negotiate with 172.17.0.1 port 60756: no matching host key type found. Their offer: ssh-rsa,ssh-dss,ecdsa-sha2-nistp256,ecdsa-sha2-nistp384,ecdsa-sha2-nistp521 [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug1: do_cleanup [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug3: PAM: sshpam_thread_cleanup entering [preauth]
Apr 15 14:44:42 n11 sshd[12282]: debug1: monitor_read_log: child log fd closed
Apr 15 14:44:42 n11 sshd[12282]: debug3: mm_request_receive: entering
Apr 15 14:44:42 n11 sshd[12282]: debug1: do_cleanup
Apr 15 14:44:42 n11 sshd[12282]: debug3: PAM: sshpam_thread_cleanup entering
Apr 15 14:44:42 n11 sshd[12282]: debug1: Killing privsep child 12283
Apr 15 14:44:42 n11 sshd[12282]: debug1: audit_event: unhandled event 12
</pre>
</details>

After enabling algorithms `rsa-sha2-512,rsa-sha2-256` with the following code, we get a different exception.
```java
        JSch.setConfig("server_host_key", JSch.getConfig("server_host_key") + ",rsa-sha2-512,rsa-sha2-256");
        JSch.setConfig("PubkeyAcceptedAlgorithms",
                JSch.getConfig("PubkeyAcceptedAlgorithms") + ",rsa-sha2-512,rsa-sha2-256");
```

```
Caused by: com.jcraft.jsch.JSchException: Session.connect: java.security.SignatureException: Signature length not correct: got 404 but was expecting 384
        at com.jcraft.jsch.Session.connect(Session.java:565)
        at com.jcraft.jsch.Session.connect(Session.java:183)
        at com.exasol.exasoltestsetup.SshConnection.createSession(SshConnection.java:39)
        ... 7 more
```

<details>
<summary>Server log</summary>
<pre>
Apr 15 14:46:04 n11 sshd[2810]: debug3: fd 5 is not O_NONBLOCK
Apr 15 14:46:04 n11 sshd[2810]: debug1: Forked child 12361.
Apr 15 14:46:04 n11 sshd[2810]: debug3: send_rexec_state: entering fd = 8 config len 3256
Apr 15 14:46:04 n11 sshd[2810]: debug3: ssh_msg_send: type 0
Apr 15 14:46:04 n11 sshd[2810]: debug3: send_rexec_state: done
Apr 15 14:46:04 n11 sshd[12361]: debug3: oom_adjust_restore
Apr 15 14:46:04 n11 sshd[12361]: debug1: Set /proc/self/oom_score_adj to 0
Apr 15 14:46:04 n11 sshd[12361]: debug1: rexec start in 5 out 5 newsock 5 pipe 7 sock 8
Apr 15 14:46:04 n11 sshd[12361]: debug1: inetd sockets after dupping: 4, 4
Apr 15 14:46:04 n11 sshd[12361]: Connection from 172.17.0.1 port 63442 on 172.17.0.3 port 22 rdomain ""
Apr 15 14:46:04 n11 sshd[12361]: debug1: Local version string SSH-2.0-OpenSSH_8.9p1 Ubuntu-3ubuntu0.6
Apr 15 14:46:04 n11 sshd[12361]: debug1: Remote protocol version 2.0, remote software version JSCH-0.1.54
Apr 15 14:46:04 n11 sshd[12361]: debug1: compat_banner: no match: JSCH-0.1.54
Apr 15 14:46:04 n11 sshd[12361]: debug2: fd 4 setting O_NONBLOCK
Apr 15 14:46:04 n11 sshd[12361]: debug3: ssh_sandbox_init: preparing seccomp filter sandbox
Apr 15 14:46:04 n11 sshd[12361]: debug2: Network child is on pid 12362
Apr 15 14:46:04 n11 sshd[12361]: debug3: preauth child monitor started
Apr 15 14:46:04 n11 sshd[12361]: debug3: privsep user:group 102:65534 [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug1: permanently_set_uid: 102/65534 [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug3: ssh_sandbox_child: setting PR_SET_NO_NEW_PRIVS [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug3: ssh_sandbox_child: attaching seccomp filter program [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug3: append_hostkey_type: ssh-rsa key not permitted by HostkeyAlgorithms [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug1: list_hostkey_types: rsa-sha2-512,rsa-sha2-256 [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug3: send packet: type 20 [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug1: SSH2_MSG_KEXINIT sent [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug3: receive packet: type 20 [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug1: SSH2_MSG_KEXINIT received [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug2: local server KEXINIT proposal [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug2: KEX algorithms: curve25519-sha256,curve25519-sha256@libssh.org,ecdh-sha2-nistp256,ecdh-sha2-nistp384,ecdh-sha2-nistp521,sntrup761x25519-sha512@openssh.com,diffie-hellman-group-exchange-sha256,diffie-hellman-group16-sha512,diffie-hellman-group18-sha512,diffie-hellman-group14-sha256,kex-strict-s-v00@openssh.com [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug2: host key algorithms: rsa-sha2-512,rsa-sha2-256 [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug2: ciphers ctos: aes128-ctr,aes192-ctr,aes256-ctr,aes128-gcm@openssh.com,aes256-gcm@openssh.com [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug2: ciphers stoc: aes128-ctr,aes192-ctr,aes256-ctr,aes128-gcm@openssh.com,aes256-gcm@openssh.com [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug2: MACs ctos: umac-64-etm@openssh.com,umac-128-etm@openssh.com,hmac-sha2-256-etm@openssh.com,hmac-sha2-512-etm@openssh.com,hmac-sha1-etm@openssh.com,umac-64@openssh.com,umac-128@openssh.com,hmac-sha2-256,hmac-sha2-512,hmac-sha1 [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug2: MACs stoc: umac-64-etm@openssh.com,umac-128-etm@openssh.com,hmac-sha2-256-etm@openssh.com,hmac-sha2-512-etm@openssh.com,hmac-sha1-etm@openssh.com,umac-64@openssh.com,umac-128@openssh.com,hmac-sha2-256,hmac-sha2-512,hmac-sha1 [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug2: compression ctos: none,zlib@openssh.com [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug2: compression stoc: none,zlib@openssh.com [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug2: languages ctos:  [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug2: languages stoc:  [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug2: first_kex_follows 0  [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug2: reserved 0  [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug2: peer client KEXINIT proposal [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug2: KEX algorithms: ecdh-sha2-nistp256,ecdh-sha2-nistp384,ecdh-sha2-nistp521,diffie-hellman-group14-sha1,diffie-hellman-group-exchange-sha256,diffie-hellman-group-exchange-sha1,diffie-hellman-group1-sha1 [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug2: host key algorithms: ssh-rsa,ssh-dss,ecdsa-sha2-nistp256,ecdsa-sha2-nistp384,ecdsa-sha2-nistp521,rsa-sha2-512,rsa-sha2-256 [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug2: ciphers ctos: aes128-ctr,aes128-cbc,3des-ctr,3des-cbc,blowfish-cbc,aes192-ctr,aes192-cbc,aes256-ctr,aes256-cbc [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug2: ciphers stoc: aes128-ctr,aes128-cbc,3des-ctr,3des-cbc,blowfish-cbc,aes192-ctr,aes192-cbc,aes256-ctr,aes256-cbc [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug2: MACs ctos: hmac-md5,hmac-sha1,hmac-sha2-256,hmac-sha1-96,hmac-md5-96 [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug2: MACs stoc: hmac-md5,hmac-sha1,hmac-sha2-256,hmac-sha1-96,hmac-md5-96 [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug2: compression ctos: none [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug2: compression stoc: none [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug2: languages ctos:  [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug2: languages stoc:  [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug2: first_kex_follows 0  [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug2: reserved 0  [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug1: kex: algorithm: ecdh-sha2-nistp256 [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug1: kex: host key algorithm: rsa-sha2-512 [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug1: kex: client->server cipher: aes128-ctr MAC: hmac-sha1 compression: none [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug1: kex: server->client cipher: aes128-ctr MAC: hmac-sha1 compression: none [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug1: expecting SSH2_MSG_KEX_ECDH_INIT [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug3: receive packet: type 30 [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug1: SSH2_MSG_KEX_ECDH_INIT received [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug3: mm_sshkey_sign: entering [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug3: mm_request_send: entering, type 6 [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug3: mm_sshkey_sign: waiting for MONITOR_ANS_SIGN [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug3: mm_request_receive_expect: entering, type 7 [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug3: mm_request_receive: entering [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug3: mm_request_receive: entering
Apr 15 14:46:04 n11 sshd[12361]: debug3: monitor_read: checking request 6
Apr 15 14:46:04 n11 sshd[12361]: debug3: mm_answer_sign: entering
Apr 15 14:46:04 n11 sshd[12361]: debug3: mm_answer_sign: rsa-sha2-512 KEX signature len=404
Apr 15 14:46:04 n11 sshd[12361]: debug3: mm_request_send: entering, type 7
Apr 15 14:46:04 n11 sshd[12361]: debug2: monitor_read: 6 used once, disabling now
Apr 15 14:46:04 n11 sshd[12361]: debug3: send packet: type 31 [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug3: send packet: type 21 [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug2: ssh_set_newkeys: mode 1 [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug1: rekey out after 4294967296 blocks [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug1: SSH2_MSG_NEWKEYS sent [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug1: expecting SSH2_MSG_NEWKEYS [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug3: receive packet: type 1 [preauth]
Apr 15 14:46:04 n11 sshd[12361]: error: Received disconnect from 172.17.0.1 port 63442:3: java.security.SignatureException: Signature length not correct: got 404 but was expecting 384 [preauth]
Apr 15 14:46:04 n11 sshd[12361]: Disconnected from 172.17.0.1 port 63442 [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug1: do_cleanup [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug3: PAM: sshpam_thread_cleanup entering [preauth]
Apr 15 14:46:04 n11 sshd[12361]: debug1: monitor_read_log: child log fd closed
Apr 15 14:46:04 n11 sshd[12361]: debug3: mm_request_receive: entering
Apr 15 14:46:04 n11 sshd[12361]: debug1: do_cleanup
Apr 15 14:46:04 n11 sshd[12361]: debug3: PAM: sshpam_thread_cleanup entering
Apr 15 14:46:04 n11 sshd[12361]: debug1: Killing privsep child 12362
Apr 15 14:46:04 n11 sshd[12361]: debug1: audit_event: unhandled event 12
</pre>
</details>

I didn't find a way to fix the signature length problem. Effort until now: 9:30h

--- a/.github/workflows/quick-checks.yml
+++ b/.github/workflows/quick-checks.yml
@@ -22,6 +22,8 @@ jobs:
     steps:
       - name: Checkout the repository
         uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
       - name: Set up JDKs
         uses: actions/setup-java@v4
         with:

--- a/.github/workflows/quick-checks.yml
+++ b/.github/workflows/quick-checks.yml
@@ -1,24 +1,27 @@
-name: Quick Checks
-
+name: CI Build Quick Checks
 on:
   push:
-    branches:
-      - main
+    branches: ["main"]
   pull_request:
 
 jobs:
-  quick-check:
+  quick-check-matrix-build:
     runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: "bash"
     permissions:
       contents: read
     concurrency:
-      group: ${{ github.workflow }}-${{ github.ref }}
+      group: ${{ github.workflow }}-${{ github.ref }}-${{ matrix.exasol_db_version }}
       cancel-in-progress: true
+    strategy:
+      fail-fast: false
+      matrix:
+        exasol_db_version: [8.26.0, 7.1.26]
     steps:
       - name: Checkout the repository
         uses: actions/checkout@v4
-        with:
-          fetch-depth: 0
       - name: Set up JDKs
         uses: actions/setup-java@v4
         with:
@@ -27,8 +30,16 @@ jobs:
             11
             17
           cache: "maven"
-      - name: "Prepare container reuse"
+      - name: Enable testcontainer reuse
+        run: echo 'testcontainers.reuse.enable=true' > "$HOME/.testcontainers.properties"
+      - name: Run tests and build with Maven
         run: |
-          echo 'testcontainers.reuse.enable=true' > "/home/runner/.testcontainers.properties"
-      - name: Run quick checks
-        run: mvn --batch-mode clean verify -DexcludedGroups=cloudOnly
+          mvn --batch-mode clean verify \
+              -Dcom.exasol.dockerdb.image=${{ matrix.exasol_db_version }} \
+              -DexcludedGroups=cloudOnly
+
+  quick-check-build:
+    needs: quick-check-matrix-build
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "Buld successful"

--- a/.project-keeper.yml
+++ b/.project-keeper.yml
@@ -7,6 +7,8 @@ sources:
 excludes:
   # Environment "aws". Will be fixed in https://github.com/exasol/project-keeper/issues/566
   - "E-PK-CORE-18: Outdated content: '.github/workflows/ci-build.yml'"
+  # Enable logging for Jsch
+  - "E-PK-CORE-18: Outdated content: 'src/test/resources/logging.properties'"
 build:
   workflows:
     - name: ci-build.yml

--- a/.vscode/settings.json
+++ b/.vscode/settings.json
@@ -14,7 +14,8 @@
   "java.sources.organizeImports.staticStarThreshold": 3,
   "java.test.config": {
     "vmArgs": [
-      "-Djava.util.logging.config.file=src/test/resources/logging.properties"
+      "-Djava.util.logging.config.file=src/test/resources/logging.properties",
+      "-Dcom.exasol.dockerdb.image=8.26.0"
     ]
   },
   "sonarlint.connectedMode.project": {

--- a/ci-isolation/pom.xml
+++ b/ci-isolation/pom.xml
@@ -38,7 +38,7 @@
         <dependency>
             <groupId>com.exasol</groupId>
             <artifactId>ci-isolation-aws</artifactId>
-            <version>2.0.2</version>
+            <version>2.0.3</version>
         </dependency>
     </dependencies>
 </project>

--- a/cloudSetup/cloudSetup.tf
+++ b/cloudSetup/cloudSetup.tf
@@ -10,5 +10,5 @@ module "exasol_setup" {
   deputy         = var.deputy
   datanode_count = 1
   project        = "ETSA"
-  exasol_image   = "Exasol-R7.1.19-BYOL"
+  exasol_image   = "Exasol-R7.1.26-BYOL"
 }

--- a/dependencies.md
+++ b/dependencies.md
@@ -11,50 +11,50 @@
 | [error-reporting-java][5]                           | [MIT License][6]                                                                                               |
 | [AWS Java SDK :: Services :: AWS CloudFormation][7] | [Apache License, Version 2.0][8]                                                                               |
 | [AWS Java SDK :: Services :: Amazon EC2][7]         | [Apache License, Version 2.0][8]                                                                               |
-| [JSch][9]                                           | [Revised BSD][10]                                                                                              |
-| [database-cleaner][11]                              | [MIT License][12]                                                                                              |
-| [Jakarta JSON Processing API][13]                   | [Eclipse Public License 2.0][14]; [GNU General Public License, version 2 with the GNU Classpath Exception][15] |
+| [JSch][9]                                           | [Revised BSD][10]; [Revised BSD][11]; [ISC][12]                                                                |
+| [database-cleaner][13]                              | [MIT License][14]                                                                                              |
+| [Jakarta JSON Processing API][15]                   | [Eclipse Public License 2.0][16]; [GNU General Public License, version 2 with the GNU Classpath Exception][17] |
 
 ## Test Dependencies
 
 | Dependency                                 | License                           |
 | ------------------------------------------ | --------------------------------- |
-| [JUnit Jupiter Engine][16]                 | [Eclipse Public License v2.0][17] |
-| [JUnit Jupiter Params][16]                 | [Eclipse Public License v2.0][17] |
-| [Hamcrest][18]                             | [BSD License 3][19]               |
-| [EqualsVerifier \| release normal jar][20] | [Apache License, Version 2.0][21] |
-| [System Lambda][22]                        | [MIT License][23]                 |
-| [SLF4J JDK14 Provider][24]                 | [MIT License][25]                 |
+| [JUnit Jupiter Engine][18]                 | [Eclipse Public License v2.0][19] |
+| [JUnit Jupiter Params][18]                 | [Eclipse Public License v2.0][19] |
+| [Hamcrest][20]                             | [BSD License 3][21]               |
+| [EqualsVerifier \| release normal jar][22] | [Apache License, Version 2.0][23] |
+| [System Lambda][24]                        | [MIT License][25]                 |
+| [SLF4J JDK14 Provider][26]                 | [MIT License][27]                 |
 
 ## Runtime Dependencies
 
 | Dependency            | License                                                                                                        |
 | --------------------- | -------------------------------------------------------------------------------------------------------------- |
-| [Eclipse Parsson][26] | [Eclipse Public License 2.0][14]; [GNU General Public License, version 2 with the GNU Classpath Exception][15] |
+| [Eclipse Parsson][28] | [Eclipse Public License 2.0][16]; [GNU General Public License, version 2 with the GNU Classpath Exception][17] |
 
 ## Plugin Dependencies
 
 | Dependency                                              | License                           |
 | ------------------------------------------------------- | --------------------------------- |
-| [SonarQube Scanner for Maven][27]                       | [GNU LGPL 3][28]                  |
-| [Apache Maven Toolchains Plugin][29]                    | [Apache License, Version 2.0][21] |
-| [Apache Maven Compiler Plugin][30]                      | [Apache-2.0][21]                  |
-| [Apache Maven Enforcer Plugin][31]                      | [Apache-2.0][21]                  |
-| [Maven Flatten Plugin][32]                              | [Apache Software Licenese][21]    |
-| [org.sonatype.ossindex.maven:ossindex-maven-plugin][33] | [ASL2][34]                        |
-| [Maven Surefire Plugin][35]                             | [Apache-2.0][21]                  |
-| [Versions Maven Plugin][36]                             | [Apache License, Version 2.0][21] |
-| [Project Keeper Maven plugin][37]                       | [The MIT License][38]             |
-| [duplicate-finder-maven-plugin Maven Mojo][39]          | [Apache License 2.0][40]          |
-| [Apache Maven Deploy Plugin][41]                        | [Apache-2.0][21]                  |
-| [Apache Maven GPG Plugin][42]                           | [Apache-2.0][21]                  |
-| [Apache Maven Source Plugin][43]                        | [Apache License, Version 2.0][21] |
-| [Apache Maven Javadoc Plugin][44]                       | [Apache-2.0][21]                  |
-| [Nexus Staging Maven Plugin][45]                        | [Eclipse Public License][46]      |
-| [Maven Failsafe Plugin][47]                             | [Apache-2.0][21]                  |
-| [JaCoCo :: Maven Plugin][48]                            | [EPL-2.0][49]                     |
-| [error-code-crawler-maven-plugin][50]                   | [MIT License][51]                 |
-| [Reproducible Build Maven Plugin][52]                   | [Apache 2.0][34]                  |
+| [SonarQube Scanner for Maven][29]                       | [GNU LGPL 3][30]                  |
+| [Apache Maven Toolchains Plugin][31]                    | [Apache License, Version 2.0][23] |
+| [Apache Maven Compiler Plugin][32]                      | [Apache-2.0][23]                  |
+| [Apache Maven Enforcer Plugin][33]                      | [Apache-2.0][23]                  |
+| [Maven Flatten Plugin][34]                              | [Apache Software Licenese][23]    |
+| [org.sonatype.ossindex.maven:ossindex-maven-plugin][35] | [ASL2][36]                        |
+| [Maven Surefire Plugin][37]                             | [Apache-2.0][23]                  |
+| [Versions Maven Plugin][38]                             | [Apache License, Version 2.0][23] |
+| [Project Keeper Maven plugin][39]                       | [The MIT License][40]             |
+| [duplicate-finder-maven-plugin Maven Mojo][41]          | [Apache License 2.0][42]          |
+| [Apache Maven Deploy Plugin][43]                        | [Apache-2.0][23]                  |
+| [Apache Maven GPG Plugin][44]                           | [Apache-2.0][23]                  |
+| [Apache Maven Source Plugin][45]                        | [Apache License, Version 2.0][23] |
+| [Apache Maven Javadoc Plugin][46]                       | [Apache-2.0][23]                  |
+| [Nexus Staging Maven Plugin][47]                        | [Eclipse Public License][48]      |
+| [Maven Failsafe Plugin][49]                             | [Apache-2.0][23]                  |
+| [JaCoCo :: Maven Plugin][50]                            | [EPL-2.0][51]                     |
+| [error-code-crawler-maven-plugin][52]                   | [MIT License][53]                 |
+| [Reproducible Build Maven Plugin][54]                   | [Apache 2.0][36]                  |
 
 [0]: https://github.com/exasol/bucketfs-java/
 [1]: https://github.com/exasol/bucketfs-java/blob/main/LICENSE
@@ -65,47 +65,49 @@
 [6]: https://github.com/exasol/error-reporting-java/blob/main/LICENSE
 [7]: https://aws.amazon.com/sdkforjava
 [8]: https://aws.amazon.com/apache2.0
-[9]: http://www.jcraft.com/jsch/
-[10]: http://www.jcraft.com/jsch/LICENSE.txt
-[11]: https://github.com/exasol/database-cleaner/
-[12]: https://github.com/exasol/database-cleaner/blob/main/LICENSE
-[13]: https://github.com/eclipse-ee4j/jsonp
-[14]: https://projects.eclipse.org/license/epl-2.0
-[15]: https://projects.eclipse.org/license/secondary-gpl-2.0-cp
-[16]: https://junit.org/junit5/
-[17]: https://www.eclipse.org/legal/epl-v20.html
-[18]: http://hamcrest.org/JavaHamcrest/
-[19]: http://opensource.org/licenses/BSD-3-Clause
-[20]: https://www.jqno.nl/equalsverifier
-[21]: https://www.apache.org/licenses/LICENSE-2.0.txt
-[22]: https://github.com/stefanbirkner/system-lambda/
-[23]: http://opensource.org/licenses/MIT
-[24]: http://www.slf4j.org
-[25]: http://www.opensource.org/licenses/mit-license.php
-[26]: https://github.com/eclipse-ee4j/parsson
-[27]: http://sonarsource.github.io/sonar-scanner-maven/
-[28]: http://www.gnu.org/licenses/lgpl.txt
-[29]: https://maven.apache.org/plugins/maven-toolchains-plugin/
-[30]: https://maven.apache.org/plugins/maven-compiler-plugin/
-[31]: https://maven.apache.org/enforcer/maven-enforcer-plugin/
-[32]: https://www.mojohaus.org/flatten-maven-plugin/
-[33]: https://sonatype.github.io/ossindex-maven/maven-plugin/
-[34]: http://www.apache.org/licenses/LICENSE-2.0.txt
-[35]: https://maven.apache.org/surefire/maven-surefire-plugin/
-[36]: https://www.mojohaus.org/versions/versions-maven-plugin/
-[37]: https://github.com/exasol/project-keeper/
-[38]: https://github.com/exasol/project-keeper/blob/main/LICENSE
-[39]: https://basepom.github.io/duplicate-finder-maven-plugin
-[40]: http://www.apache.org/licenses/LICENSE-2.0.html
-[41]: https://maven.apache.org/plugins/maven-deploy-plugin/
-[42]: https://maven.apache.org/plugins/maven-gpg-plugin/
-[43]: https://maven.apache.org/plugins/maven-source-plugin/
-[44]: https://maven.apache.org/plugins/maven-javadoc-plugin/
-[45]: http://www.sonatype.com/public-parent/nexus-maven-plugins/nexus-staging/nexus-staging-maven-plugin/
-[46]: http://www.eclipse.org/legal/epl-v10.html
-[47]: https://maven.apache.org/surefire/maven-failsafe-plugin/
-[48]: https://www.jacoco.org/jacoco/trunk/doc/maven.html
-[49]: https://www.eclipse.org/legal/epl-2.0/
-[50]: https://github.com/exasol/error-code-crawler-maven-plugin/
-[51]: https://github.com/exasol/error-code-crawler-maven-plugin/blob/main/LICENSE
-[52]: http://zlika.github.io/reproducible-build-maven-plugin
+[9]: https://github.com/mwiede/jsch
+[10]: https://github.com/mwiede/jsch/blob/master/LICENSE.txt
+[11]: https://github.com/mwiede/jsch/blob/master/LICENSE.JZlib.txt
+[12]: https://github.com/mwiede/jsch/blob/master/LICENSE.jBCrypt.txt
+[13]: https://github.com/exasol/database-cleaner/
+[14]: https://github.com/exasol/database-cleaner/blob/main/LICENSE
+[15]: https://github.com/eclipse-ee4j/jsonp
+[16]: https://projects.eclipse.org/license/epl-2.0
+[17]: https://projects.eclipse.org/license/secondary-gpl-2.0-cp
+[18]: https://junit.org/junit5/
+[19]: https://www.eclipse.org/legal/epl-v20.html
+[20]: http://hamcrest.org/JavaHamcrest/
+[21]: http://opensource.org/licenses/BSD-3-Clause
+[22]: https://www.jqno.nl/equalsverifier
+[23]: https://www.apache.org/licenses/LICENSE-2.0.txt
+[24]: https://github.com/stefanbirkner/system-lambda/
+[25]: http://opensource.org/licenses/MIT
+[26]: http://www.slf4j.org
+[27]: http://www.opensource.org/licenses/mit-license.php
+[28]: https://github.com/eclipse-ee4j/parsson
+[29]: http://sonarsource.github.io/sonar-scanner-maven/
+[30]: http://www.gnu.org/licenses/lgpl.txt
+[31]: https://maven.apache.org/plugins/maven-toolchains-plugin/
+[32]: https://maven.apache.org/plugins/maven-compiler-plugin/
+[33]: https://maven.apache.org/enforcer/maven-enforcer-plugin/
+[34]: https://www.mojohaus.org/flatten-maven-plugin/
+[35]: https://sonatype.github.io/ossindex-maven/maven-plugin/
+[36]: http://www.apache.org/licenses/LICENSE-2.0.txt
+[37]: https://maven.apache.org/surefire/maven-surefire-plugin/
+[38]: https://www.mojohaus.org/versions/versions-maven-plugin/
+[39]: https://github.com/exasol/project-keeper/
+[40]: https://github.com/exasol/project-keeper/blob/main/LICENSE
+[41]: https://basepom.github.io/duplicate-finder-maven-plugin
+[42]: http://www.apache.org/licenses/LICENSE-2.0.html
+[43]: https://maven.apache.org/plugins/maven-deploy-plugin/
+[44]: https://maven.apache.org/plugins/maven-gpg-plugin/
+[45]: https://maven.apache.org/plugins/maven-source-plugin/
+[46]: https://maven.apache.org/plugins/maven-javadoc-plugin/
+[47]: http://www.sonatype.com/public-parent/nexus-maven-plugins/nexus-staging/nexus-staging-maven-plugin/
+[48]: http://www.eclipse.org/legal/epl-v10.html
+[49]: https://maven.apache.org/surefire/maven-failsafe-plugin/
+[50]: https://www.jacoco.org/jacoco/trunk/doc/maven.html
+[51]: https://www.eclipse.org/legal/epl-2.0/
+[52]: https://github.com/exasol/error-code-crawler-maven-plugin/
+[53]: https://github.com/exasol/error-code-crawler-maven-plugin/blob/main/LICENSE
+[54]: http://zlika.github.io/reproducible-build-maven-plugin

--- a/doc/changes/changelog.md
+++ b/doc/changes/changelog.md
@@ -1,5 +1,6 @@
 # Changes
 
+* [2.1.3](changes_2.1.3.md)
 * [2.1.2](changes_2.1.2.md)
 * [2.1.1](changes_2.1.1.md)
 * [2.1.0](changes_2.1.0.md)

--- a/doc/changes/changes_2.1.3.md
+++ b/doc/changes/changes_2.1.3.md
@@ -1,0 +1,16 @@
+# Exasol Test Setup Abstraction Java 2.1.3, released 2024-??-??
+
+Code name:
+
+## Summary
+
+## Features
+
+* ISSUE_NUMBER: description
+
+## Dependency Updates
+
+### Compile Dependency Updates
+
+* Updated `software.amazon.awssdk:cloudformation:2.25.26` to `2.25.29`
+* Updated `software.amazon.awssdk:ec2:2.25.26` to `2.25.29`

--- a/doc/changes/changes_2.1.3.md
+++ b/doc/changes/changes_2.1.3.md
@@ -12,5 +12,11 @@ Code name:
 
 ### Compile Dependency Updates
 
-* Updated `software.amazon.awssdk:cloudformation:2.25.26` to `2.25.29`
-* Updated `software.amazon.awssdk:ec2:2.25.26` to `2.25.29`
+* Added `com.github.mwiede:jsch:0.2.17`
+* Removed `com.jcraft:jsch:0.1.55`
+* Updated `software.amazon.awssdk:cloudformation:2.25.26` to `2.25.31`
+* Updated `software.amazon.awssdk:ec2:2.25.26` to `2.25.31`
+
+### Test Dependency Updates
+
+* Updated `org.slf4j:slf4j-jdk14:2.0.12` to `2.0.13`

--- a/doc/changes/changes_2.1.3.md
+++ b/doc/changes/changes_2.1.3.md
@@ -1,12 +1,14 @@
-# Exasol Test Setup Abstraction Java 2.1.3, released 2024-??-??
+# Exasol Test Setup Abstraction Java 2.1.3, released 2024-04-16
 
-Code name:
+Code name: Fix SSH connection to Exasol Docker container 8.25.0 and later
 
 ## Summary
 
-## Features
+This release fixes error `JSchException: Algorithm negotiation fail` when starting an Exasol Docker container 8.25.0 and later.
 
-* ISSUE_NUMBER: description
+## Bugfixes
+
+* #75: Fixed SSH connection to Exasol Docker container 8.25.0 and later
 
 ## Dependency Updates
 

--- a/pk_generated_parent.pom
+++ b/pk_generated_parent.pom
@@ -3,7 +3,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.exasol</groupId>
     <artifactId>exasol-test-setup-abstraction-java-generated-parent</artifactId>
-    <version>2.1.2</version>
+    <version>2.1.3</version>
     <packaging>pom</packaging>
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>

--- a/pom.xml
+++ b/pom.xml
@@ -2,13 +2,13 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <artifactId>exasol-test-setup-abstraction-java</artifactId>
-    <version>2.1.2</version>
+    <version>2.1.3</version>
     <name>exasol-test-setup-abstraction-java</name>
     <description>Common interface for testing with an exasol database local and on AWS using Java.</description>
     <url>https://github.com/exasol/exasol-test-setup-abstraction-java/</url>
     <properties>
         <junit.version>5.10.2</junit.version>
-        <awssdk.version>2.25.26</awssdk.version>
+        <awssdk.version>2.25.29</awssdk.version>
     </properties>
     <dependencyManagement>
         <dependencies>
@@ -171,7 +171,6 @@
                     <checkCompileClasspath>true</checkCompileClasspath>
                     <checkRuntimeClasspath>true</checkRuntimeClasspath>
                     <checkTestClasspath>false</checkTestClasspath>
-                    <quiet>true</quiet>
                     <preferLocal>true</preferLocal>
                     <useResultFile>false</useResultFile>
                 </configuration>
@@ -181,7 +180,7 @@
     <parent>
         <artifactId>exasol-test-setup-abstraction-java-generated-parent</artifactId>
         <groupId>com.exasol</groupId>
-        <version>2.1.2</version>
+        <version>2.1.3</version>
         <relativePath>pk_generated_parent.pom</relativePath>
     </parent>
 </project>

--- a/pom.xml
+++ b/pom.xml
@@ -31,6 +31,13 @@
             <groupId>com.exasol</groupId>
             <artifactId>exasol-testcontainers</artifactId>
             <version>7.0.1</version>
+            <exclusions>
+                <exclusion>
+                    <!-- We use com.github.mwiede:jsch instead which contains duplicate classes -->
+                    <groupId>com.jcraft</groupId>
+                    <artifactId>jsch</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
         <!-- Required for ExaOperationGateway -->
         <dependency>

--- a/pom.xml
+++ b/pom.xml
@@ -8,7 +8,7 @@
     <url>https://github.com/exasol/exasol-test-setup-abstraction-java/</url>
     <properties>
         <junit.version>5.10.2</junit.version>
-        <awssdk.version>2.25.29</awssdk.version>
+        <awssdk.version>2.25.31</awssdk.version>
     </properties>
     <dependencyManagement>
         <dependencies>
@@ -54,9 +54,9 @@
             <version>${awssdk.version}</version>
         </dependency>
         <dependency>
-            <groupId>com.jcraft</groupId>
+            <groupId>com.github.mwiede</groupId>
             <artifactId>jsch</artifactId>
-            <version>0.1.55</version>
+            <version>0.2.17</version>
         </dependency>
         <dependency>
             <groupId>com.exasol</groupId>
@@ -109,7 +109,7 @@
             <!-- Enable log output from testcontainers in integration tests -->
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-jdk14</artifactId>
-            <version>2.0.12</version>
+            <version>2.0.13</version>
             <scope>test</scope>
         </dependency>
     </dependencies>

--- a/src/main/java/com/exasol/exasoltestsetup/SessionBuilder.java
+++ b/src/main/java/com/exasol/exasoltestsetup/SessionBuilder.java
@@ -1,6 +1,7 @@
 package com.exasol.exasoltestsetup;
 
 import java.util.*;
+import java.util.logging.Logger;
 
 import com.exasol.exasoltestsetup.identity.IdentityProvider;
 import com.jcraft.jsch.*;
@@ -9,6 +10,7 @@ import com.jcraft.jsch.*;
  * Builder for {@link com.jcraft.jsch.Session}
  */
 public class SessionBuilder {
+    private static final Logger LOG = Logger.getLogger(SessionBuilder.class.getName());
     private String user;
     private String host;
     private int port;
@@ -75,9 +77,11 @@ public class SessionBuilder {
      * @throws JSchException if session creation fails
      */
     public Session build() throws JSchException {
+        JSch.setLogger(new JulLogger());
         final JSch jsch = new JSch();
         this.identityProvider.addIdentityTo(jsch);
         final Session session = jsch.getSession(this.user, this.host, this.port);
+        LOG.fine(() -> "Connecting to via ssh: ssh " + this.user + "@" + this.host + " -p " + this.port);
         session.setConfig(new Hashtable<>(this.config));
         return session;
     }

--- a/src/main/java/com/exasol/exasoltestsetup/SessionBuilder.java
+++ b/src/main/java/com/exasol/exasoltestsetup/SessionBuilder.java
@@ -1,6 +1,7 @@
 package com.exasol.exasoltestsetup;
 
 import java.util.*;
+import java.util.logging.Level;
 import java.util.logging.Logger;
 
 import com.exasol.exasoltestsetup.identity.IdentityProvider;
@@ -77,11 +78,13 @@ public class SessionBuilder {
      * @throws JSchException if session creation fails
      */
     public Session build() throws JSchException {
-        JSch.setLogger(new JulLogger());
+        if (LOG.isLoggable(Level.FINEST)) {
+            JSch.setLogger(new JulLogger());
+        }
         final JSch jsch = new JSch();
         this.identityProvider.addIdentityTo(jsch);
-        final Session session = jsch.getSession(this.user, this.host, this.port);
         LOG.fine(() -> "Connecting to via ssh: ssh " + this.user + "@" + this.host + " -p " + this.port);
+        final Session session = jsch.getSession(this.user, this.host, this.port);
         session.setConfig(new Hashtable<>(this.config));
         return session;
     }

--- a/src/main/java/com/exasol/exasoltestsetup/testcontainers/ExasolTestcontainerTestSetup.java
+++ b/src/main/java/com/exasol/exasoltestsetup/testcontainers/ExasolTestcontainerTestSetup.java
@@ -22,7 +22,7 @@ import com.jcraft.jsch.*;
  */
 public class ExasolTestcontainerTestSetup implements ExasolTestSetup {
     private static final int SSH_PORT = 22;
-    private final ExasolContainer<? extends ExasolContainer<?>> exasolContainer = new ExasolContainer<>("7.1.25")
+    private final ExasolContainer<? extends ExasolContainer<?>> exasolContainer = new ExasolContainer<>("7.1.26")
             .withReuse(true);
     private final SshConnection sshConnection;
     private final KeyPair keyPair;

--- a/src/main/java/com/exasol/exasoltestsetup/testcontainers/ExasolTestcontainerTestSetup.java
+++ b/src/main/java/com/exasol/exasoltestsetup/testcontainers/ExasolTestcontainerTestSetup.java
@@ -33,14 +33,18 @@ public class ExasolTestcontainerTestSetup implements ExasolTestSetup {
     public ExasolTestcontainerTestSetup() {
         this.exasolContainer.addExposedPort(SSH_PORT);
         this.exasolContainer.start();
+        this.keyPair = generateKeyPair();
+        installSshKeyInDatabase();
+        this.sshConnection = new SshConnection(sessionBuilder());
+    }
+
+    private static KeyPair generateKeyPair() {
         try {
-            this.keyPair = KeyPair.genKeyPair(new JSch(), KeyPair.RSA);
+            return KeyPair.genKeyPair(new JSch(), KeyPair.RSA, 3072);
         } catch (final JSchException exception) {
             throw new IllegalStateException(ExaError.messageBuilder("F-ETAJ-36")
                     .message("Failed to generate temporary ssh-key.").ticketMitigation().toString(), exception);
         }
-        installSshKeyInDatabase();
-        this.sshConnection = new SshConnection(sessionBuilder());
     }
 
     @Override
@@ -73,9 +77,10 @@ public class ExasolTestcontainerTestSetup implements ExasolTestSetup {
 
     private void installSshKeyInDatabase() {
         try {
+            final String publicKey = Base64.getEncoder().encodeToString(this.keyPair.getPublicKeyBlob());
+            final String keyType = "ssh-rsa";
             runInContainerWithCheck("bash", "-c",
-                    "echo \"ssh-rsa " + Base64.getEncoder().encodeToString(this.keyPair.getPublicKeyBlob())
-                            + "\" >> /root/.ssh/authorized_keys");
+                    "echo '" + keyType + " " + publicKey + "' >> /root/.ssh/authorized_keys");
         } catch (final IOException exception) {
             throw new IllegalStateException(ExaError.messageBuilder("F-ETAJ-16")
                     .message("Failed to upload ssh-public-key to container. This is required for login in via SSH.")

--- a/src/test/resources/logging.properties
+++ b/src/test/resources/logging.properties
@@ -4,3 +4,4 @@ java.util.logging.ConsoleHandler.level=ALL
 java.util.logging.ConsoleHandler.formatter=java.util.logging.SimpleFormatter
 java.util.logging.SimpleFormatter.format=%1$tF %1$tT.%1$tL [%4$-7s] %5$s %n
 com.exasol.level=ALL
+com.jcraft.jsch.level=ALL

